builtin/game/item.lua

```diff
@@ -579,7 +579,7 @@ function core.node_dig(pos, node, digger)
 		local tp = wielded:get_tool_capabilities()
 		local dp = core.get_dig_params(def and def.groups, tp)
 		if wdef and wdef.after_use then
-			wielded = wdef.after_use(wielded, digger, node, dp) or wielded
+			wielded = wdef.after_use(wielded, digger, node, dp, old_pt) or wielded
```

Comment:
What would speak about giving this new parameter also to other callbacks, like `can_dig`?

Reply:
I just implemented it for the one callback where I needed it because I wasn't sure if it would be wanted at all. I can look into where else it would make sense to pass the new param (and it is possible to do so without major code changes).